### PR TITLE
Add profile image upload fuction

### PR DESCRIPTION
**Pull request by YiShianLiu**

### CODE REVIEW

**Review by sarinalalria — APPROVED** [review]
Tested the changes and uploading image works as intended. good job  Maybe add a max height to the image instead of fitting whatever the size image is because if the image is too big, then it is going to show up too big on the side bar, but overall this looks good.

File: HTML/JavaScript/dashboard.js [hunk 1]
```diff
@@ -2,4 +2,22 @@ $(document).ready(function () {
...
-});
\ No newline at end of file
+});
+function clickPicture(){
+    let file = document.getElementById("file")
+    file.click()
+}
+function previewFile(){
+    console.log("debug1")
+    var preview = document.querySelector('img');
```
Comment (zeshengchen) [inline HTML/JavaScript/dashboard.js:11]: Adding ";" at the end of line or not is fine. But please use one style!

File: HTML/JavaScript/dashboard.js [hunk 2]
```diff
@@ -2,4 +2,22 @@ $(document).ready(function () {
...
     });
-});
\ No newline at end of file
+});
+function clickPicture(){
+    let file = document.getElementById("file")
+    file.click()
+}
+function previewFile(){
+    console.log("debug1")
```
Comment (zeshengchen) [inline HTML/JavaScript/dashboard.js]: If it is just for debugging, please remove it.

**Review by zeshengchen — APPROVED** [review]
See my two comments!

**Review by nscheiber — APPROVED** [review]

**Review by Sinclair198 — APPROVED** [review]

**Pull request merged**